runtime/php/php.Dockerfile

```diff
@@ -413,10 +445,22 @@ RUN set -xe; \
  && make \
  && make install
 
+RUN set -xe; \
+    mkdir -p /opt/bref/etc/php/config.d \
+ && echo  zend_extension=opcache.so > /opt/bref/etc/php/config.d/opcache.ini \
+ && echo display_errors=0 >> /opt/bref/etc/php/config.d/opcache.ini \
+ && echo dopcache.enable=1 >> /opt/bref/etc/php/config.d/opcache.ini \
+ && echo dopcache.validate_permission=0 >> /opt/bref/etc/php/config.d/opcache.ini \
+ && echo dopcache.validate_timestamps=0 >> /opt/bref/etc/php/config.d/opcache.ini \
+ && echo dpcache.memory_consumption=128 >> /opt/bref/etc/php/config.d/opcache.ini \
+ && echo dopcache.max_accelerated_files=10000 >> /opt/bref/etc/php/config.d/opcache.ini
```

Comment:
These flags are already customized in [our php.ini](https://github.com/mnapoli/bref/blob/0.3/runtime/php/layers/function/php.ini) right? (by the way there are typos I think: `dopcache` -> `opcache`)

I think it's best to keep Bref's customization/optimization config separately (in the existing layer files). For example it's different between PHP CLI and FPM. And at least Bref's optimizations are easily identified.

Reply:
You want to just pull that out then?

Reply:
Yep :+1:

In any case this will change with #191, probably the best would be to have:

/etc/bref/php/php.ini -> default production php.ini from the PHP distribution
/etc/bref/php/conf.d/ext-*.ini -> default config files from extensions (actually I don't know if there are any here)
/etc/bref/php/conf.d/bref.ini -> Bref's optimized config for Lambda (which is currently here https://github.com/mnapoli/bref/blob/0.3/runtime/php/layers/function/php.ini)

That way it's more explicit.